src/main.rs

```diff
@@ -1,61 +1,7 @@
-use atrium_api::{record::KnownRecord::AppBskyFeedPost, types::string};
-use clap::Parser;
-use jetstream_oxide::{
-    events::{commit::CommitEvent, JetstreamEvent::Commit},
-    DefaultJetstreamEndpoints, JetstreamCompression, JetstreamConfig, JetstreamConnector,
-};
-
-#[derive(Parser, Debug)]
-#[command(version, about, long_about = None)]
-struct Args {
-    /// The DIDs to listen for events on, if not provided we will listen for all DIDs.
-    #[arg(short, long)]
-    did: Option<Vec<string::Did>>,
-    /// The NSID for the collection to listen for (e.g. `app.bsky.feed.post`).
-    #[arg(short, long)]
-    nsid: string::Nsid,
-}
-
 #[tokio::main]
-async fn main() -> anyhow::Result<()> {
-    let args = Args::parse();
-
-    let dids = args.did.unwrap_or_default();
-    let config = JetstreamConfig {
-        endpoint: DefaultJetstreamEndpoints::USEastOne.into(),
-        wanted_collections: vec![args.nsid.clone()],
-        wanted_dids: dids.clone(),
-        compression: JetstreamCompression::Zstd,
-        cursor: None,
-    };
-
-    let jetstream = JetstreamConnector::new(config)?;
-    let (receiver, _) = jetstream.connect().await?;
-
-    println!(
-        "Listening for '{}' events on DIDs: {:?}",
-        args.nsid.as_str(),
-        dids,
-    );
-
-    while let Ok(event) = receiver.recv_async().await {
-        if let Commit(commit) = event {
-            match commit {
-                CommitEvent::Create { info: _, commit } => {
-                    if let AppBskyFeedPost(record) = commit.record {
-                        println!(
-                            "New post created! ({})\n\n'{}'",
-                            commit.info.rkey, record.text
-                        );
-                    }
-                }
-                CommitEvent::Delete { info: _, commit } => {
-                    println!("A post has been deleted. ({})", commit.rkey);
-                }
-                _ => {}
-            }
-        }
-    }
-
+async fn main() -> color_eyre::Result<()> {
+    let router = atbb::run().await?;
+    let listener = tokio::net::TcpListener::bind("0.0.0.0:8080").await.unwrap();
```

Comment:
*todo*: I'm sure you already thought the same, but we'll want this to be configurable, maybe through environment variables, later on.

Reply:
I just pushed up a change so that we can configure the bind port/addr.

Reply:
Looks great!